main.html

```diff
@@ -5,18 +5,104 @@
 		<meta charset="UTF-8">
 		<title>pokedex</title>
 		<meta name="viewport" content="width=device-width, initial-scale=1">
-		<!-- make sure to link to your css file -->
+		<link rel="stylesheet" type="text/css" href="main.css">
 	</head>
 
 	<body>
-		<!-- include a header tag -->
+		<header>
+      <div id=header-navigation>
+        <button>Dropdown
+          </button>
+        </div>
+			<h3 id="header-title">PokeDex</h3>
+      <div id=header-navigation><img src="assets/icons/pokeball_icon.png"></div>
+		</header>
+
 
 		<!-- depending on how you structure your html, you should have 3 section tags -->
 			<!-- 1) search box -->
+		<div id="search-bar">Search Pokemon
+	  	</div>
 			<!-- 2) selected pokemon: picture, property and description of pikachu -->
-			<!-- 3) other pokemon pictures in 2x6 format -->
-
+    <section>
+      <div>
+        <div id="pokemon-pikachu">
+        	<img src="assets/images/pikachu.png"> 
+        </div>
+		<div id="pokemon-description">
+			<div id="pokemon-number">
+				<h2>No. 001 Pikachu</h2>
+				</div>
+			<div id="pokemon-detail">
+				<p>- Electric type</p>
+				<p>- Gentle temperment</p>
+				<p>- Grasslands</p>
```

Comment:
This would have been an appropriate case to use a list (ol or ul) instead of div and paragraph elements.